src/firmware/main/math/polynomial_1d_test.cpp

```diff
@@ -0,0 +1,74 @@
+extern "C"
+{
+#include "firmware/main/math/polynomial_1d.h"
+}
+
+#include <gtest/gtest.h>
+
+class Polynomial1dTest : public testing::Test
```

Comment:
Why is there a test fixture if `setup` and `teardown` aren't used?

Reply:
Just shared functions.

Reply:
I don't see any shared functions in the tests for Polynomial1D, only for Polynomial2D

Reply:
Quite right, removed the test fixture and made them independent tests.